packages/react/src/components/Nav/Nav.types.ts

```diff
@@ -167,6 +167,9 @@ export interface INavLinkGroup {
 
   /**
    * ARIA label when group is collapsed and can be expanded.
+   * WARNING: using separate labels for expanded and collapsed state is not recommended.
+   *
+   * @deprecated Use `expandAriaLabel` instead.
```

Comment:
_nit:_ Make the usage more explicit

```suggestion
   * @deprecated Use `expandAriaLabel` on its own instead.
```